spec/features/search/case_reference_spec.rb

```diff
@@ -10,7 +10,7 @@
   scenario 'with results', stub_case_reference_results: true do
     visit '/'
 
-    choose 'By case reference'
+    choose 'Search by case reference'
```

Comment:
Maybe use key from locales, therefore no need to update spec when locales value changes

Reply:
For me the test actually tests the translation is working too